xilem/examples/calc.rs

```diff
@@ -293,13 +292,19 @@ fn run(event_loop: EventLoopBuilder) -> Result<(), EventLoopError> {
     };
 
     let app = Xilem::new(data, app_logic);
-    let min_window_size = LogicalSize::new(200., 200.);
-    let window_size = LogicalSize::new(400., 500.);
-    let window_attributes = Window::default_attributes()
-        .with_title("Calculator")
-        .with_resizable(true)
-        .with_min_inner_size(min_window_size)
-        .with_inner_size(window_size);
+    #[cfg(not(any(target_os = "ios", target_os = "android")))]
```

Comment:
This example works fine on my Android device - I believe that this is instead a winit bug on iOS